app/controllers/api/v1/rents_controller.rb

```diff
@@ -1,13 +1,15 @@
 module Api
   module V1
     class RentsController < ApiController
+      include Wor::Paginate
```

Comment:
Remember, we added this to `ApiController`

Reply:
:man_facepalming: